src/spyglass/common/common_task.py

```diff
@@ -168,14 +178,64 @@ def make(self, key):
 
                     key["epoch"] = epoch
                     target_interval = str(epoch).zfill(2)
+                    target_interval_name = None
                     for interval in session_intervals:
                         if (
                             target_interval in interval
                         ):  # TODO this is not true for the beans file
+                            target_interval_name = interval
                             break
+                    if target_interval_name is None:
+                        logger.warn(
+                            f"Interval not found for epoch {epoch} in {nwb_file_name}"
+                        )
+                        continue
                     # TODO case when interval is not found is not handled
-                    key["interval_list_name"] = interval
+                    key["interval_list_name"] = target_interval_name
                     task_inserts.append(key.copy())
+
+        # Add tasks from config
+        for task in config_tasks:
+            new_key = key.copy()
+            new_key["task_name"] = task.get("task_name")
+            camera_ids = task.get("camera_id", [])
+            valid_camera_ids = [
+                camera_id
+                for camera_id in camera_ids
+                if camera_id in camera_names.keys()
+            ]
+            if valid_camera_ids:
+                key["camera_names"] = [
```

Comment:
Thanks for the wrong key update catch.  

This section is matched to the one inserting from nwb files above. I would lean towards keeping them the same rather than change both to keep the diffs cleaner